packages/database/src/migrations/20210602005250-AddStriveResistanceOverlay-modifies-data.js

```diff
@@ -0,0 +1,94 @@
+('use strict');
+
+import { generateId, insertObject } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const mapOverlay = {
+  id: 'PNG_STRIVE_Resistence',
```

Comment:
```suggestion
  id: 'PNG_STRIVE_Resistance',
```

Reply:
good catch! spelling is not my strength..